src/Combinatorics/Graphs/functions.jl

```diff
@@ -788,6 +788,35 @@ false
 """
 is_isomorphic(g1::Graph{T}, g2::Graph{T}) where {T <: Union{Directed, Undirected}} = Polymake.graph.isomorphic(pm_object(g1), pm_object(g2))::Bool
 
+
+function is_isomorphic_with_map(G1::Graph, G2::Graph)
```

Comment:
The second return value is not a map, so this name is seems misleading to me

Reply:
I support @fieker 's suggestions of is_isomorphic_with_data for the comparison of the graphs.
This suggests to the user that one should look at the documentation to find out, what the data is.